modules/sagemaker/sagemaker-groundtruth/deployspec.yaml

```diff
@@ -0,0 +1,38 @@
+publishGenericEnvVariables: true
+deploy:
+  phases:
+    install:
+      commands:
+      - npm install -g aws-cdk@2.137.0
+      - pip install -r requirements.txt
+    pre_build:
+      commands:
+      - echo "Prebuild stage"
+    build:
+      commands:
+       - cdk deploy --all --require-approval never --progress events --app "python app.py" --outputs-file ./cdk-exports.json
+       - cat cdk-exports.json
+       # Export metadata
+       - seedfarmer metadata convert -f cdk-exports.json || true
+       - export SEEDFARMER_MODULE_METADATA=$(cat SEEDFARMER_MODULE_METADATA)
+       - echo "bash deploy.sh"
```

Comment:
Was this also for debug? I don't see a `deploy.sh` and we don't seem to usually export the `SEEDFARMER_MODULE_METADATA`? (`sagemaker-studio` module does this, but I'm not sure why.)